docusaurus/docs/Android/01-basics/03-getting-started.mdx

```diff
@@ -38,22 +38,27 @@ class MainActivity : AppCompatActivity() {
 }
 ```
 
-### Creating a ChatDomain
+### Adding an Offline Plugin
 
-If you want to have offline support or use the UI Components package, you'll need to initialize the `ChatDomain` class in addition to `ChatClient`. You can skip this initialization step if you're only using the low-level client.
+If you want to have offline support or **use the UI Components package**, you'll need to initialize the `OfflinePlugin` class and add it to the `ChatClient.Builder`. You can skip this initialization step if you're only using the low-level client.
 
-The initialization is done similarly to `ChatClient`, using a builder, which exposes several methods for configuring the SDK's behavior:
+The initialization should be done using `StreamOfflinePluginFactory`:
 
 ```kotlin
-val chatClient = ChatClient.Builder(apiKey, appContext).build()
+val offlinePluginFactory = StreamOfflinePluginFactory(
```

Comment:
It might be worth to note here after the examples (I imagine it's covered in offline support page) how each boolean flag enables or disables one of the features.

Just a small note, so that people don't automatically think they _have to_ enable it